cloud_dashboard/src/molecules/D3MetricsBlock.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import EntityInfoRecordData from 'model/EntityInfoRecordData';
+import D3HorizonChart from 'atoms/D3HorizonChart';
+
+const D3MetricsBlock = ({ record }: { record: EntityInfoRecordData }) => {
+  if (record.type !== 'metrics') {
+    return <></>;
+  }
+
+  return <>
+    {
+      record.record.map((data, index) => {
+        return <div className="row">
+          <div className="col" key={index}>
+            <D3HorizonChart id={`metric-${index}`} dataSet={
+              data.record.map((r) => {
+                return [new Date(r.x * 1000), r.y];
+              })
+            }
+              height={100} titleFontSize={10} titleColor="white" title={data.title}
+              colors={
+                index === 0
+                  ? ['#edf8e9', '#bae4b3', '#74c476', '#31a354', '#006d2c']
+                  : ['#e9f8f4', '#b3e4dc', '#74c1c4', '#317fa3', '#003f6d']
+              } />
```

Comment:
@RyoYamashita765 This can be reduce one indent?

```suggestion
            height={100} titleFontSize={10} titleColor="white" title={data.title}
            colors={
              index === 0
                ? ['#edf8e9', '#bae4b3', '#74c476', '#31a354', '#006d2c']
                : ['#e9f8f4', '#b3e4dc', '#74c1c4', '#317fa3', '#003f6d']
            } />
```

Reply:
@naoi VSCode's Formatter process indents attribute descriptions by one level relative to the parentheses of the tag. I committed this because I thought it was fine, but do I really need to align the indentation to the same height as the tag?

```typescript
            // tag indent line
            <D3HorizonChart id={`metric-${index}`} dataSet={
              data.record.map((r) => {
                return [new Date(r.x * 1000), r.y];
              })
            }
              // attributes' indent line
              height={100} titleFontSize={10} titleColor="white" title={data.title}
              colors={
                index === 0
                  ? ['#edf8e9', '#bae4b3', '#74c476', '#31a354', '#006d2c']
                  : ['#e9f8f4', '#b3e4dc', '#74c1c4', '#317fa3', '#003f6d']
              } />
```

However, it doesn't look good if the indentation of the parentheses is different between the colors attribute and the dataSet attribute, so I will submit it with the same indentation.

```typescript
            <D3HorizonChart
              id={`metric-${index}`}
              dataSet={
                data.record.map((r) => {
                  return [new Date(r.x * 1000), r.y];
                })
              }
              height={100}
              titleFontSize={10}
              titleColor="white"
              title={data.title}
              colors={
                index === 0
                  ? ['#edf8e9', '#bae4b3', '#74c476', '#31a354', '#006d2c']
                  : ['#e9f8f4', '#b3e4dc', '#74c1c4', '#317fa3', '#003f6d']
              } />
```

Reply:
@RyoYamashita765 OK, Thanks!